fluent-bundle/src/resolver/pattern.rs

```diff
@@ -91,13 +91,33 @@ impl<'bundle> ResolveValue<'bundle> for ast::Pattern<&'bundle str> {
     {
         let len = self.elements.len();
 
+        // more than 1 element means concatenation, which is more efficient to write to a String
+        // 1 element often means just a message reference that can be passed back as a Cow::Borrowed
         if len == 1 {
```

Comment:
Suggestion: I would invert this statement to be about the case below it. Something like:

> If there is only 1 element, than it is more efficient to attempt to resolve as a message reference that can be passed back as a Cow::Borrowed rather than writing a new String.

As it is, it's a little confusing to read, since it is more about the case _below_ the if statement.